packages/salesforcedx-vscode-core/test/vscode-integration/commands/auth/forceAuthAccessTokenLogin.test.ts

```diff
@@ -21,9 +21,7 @@ describe('Force Auth Access Token Login', () => {
   let workspaceCheckerStub: SinonStub;
   let accessTokenParamsGathererStub: SinonStub;
   let authInfoCreateStub: SinonStub;
-  let authInfoSaveStub: SinonStub;
-  let authInfoSetAliasStub: SinonStub;
-  let authInfoSetAsDefaultStub: SinonStub;
+  let handleAliasAndDefaultSettingsStub: SinonStub;
```

Comment:
The command changed to use the above method that I'm stubbing instead of calling the individual methods.